src/components/Header/Header.styled.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+
+const StyledHeader = styled.div`
```

Comment:
Wouldn't \<header> be better than just \<div> ? 🤔

Reply:
yes sure it. adjusted accordingly :)